### ISSUE TITLE
Учитываются ли "параллельные" чанки?

### ISSUE THREAD

**Issue by phant** [Alex-FIR-IT/Loader_for_webinar.ru#3]
Хочется уточнить, учитывается ли то что чанки могут быть параллельно идущими в трансляции?
Например, 2 ведущих одновременно говорят и при этом показывается рабочий стол.

Судя по коду в files_merging.py как минимум звук из файлов склеиваются по порядку, без учета "параллелизма". 
В итоге получится нечто странное..
https://github.com/Alex-FIR-IT/Loader_for_webinar.ru/blob/3e68ca1016fb43b6c1624afd1e5bd0e82620775a/scripts/files_merging.py#L105-L110

**Comment by Alex-FIR-IT** [comment]
Да, если будет 2 ведущих и их звуковые дорожки будут находится в разных аудио файлах, то тогда они склеются по порядку, что не является ожидаемым результатом. Нужно будет протестировать такой вариант и дорабатывать скрипт в случае, если аудио дорожки участников трансляции пишутся независимо друг от друга
